src/main/java/com/epam/aidial/core/controller/BucketController.java

```diff
@@ -19,7 +19,14 @@ public Future<?> getBucket() {
         EncryptionService encryptionService = proxy.getEncryptionService();
         String bucketLocation = BlobStorageUtil.buildUserBucket(context);
         String encryptedBucket = encryptionService.encrypt(bucketLocation);
-
-        return context.respond(HttpStatus.OK, new Bucket(encryptedBucket));
+        String appDataBucket = BlobStorageUtil.buildAppDataBucket(context);
+        String appDataLocation;
+        if (appDataBucket == null) {
+            appDataLocation = null;
+        } else {
+            String encryptedAppDataBucket = encryptionService.encrypt(bucketLocation);
+            appDataLocation = encryptedAppDataBucket + String.format(BlobStorageUtil.APPDATA_PATTERN.formatted(context.getSourceDeployment()));
```

Comment:
missing path separator between encrypted bucket and "appdata..."

Reply:
fixed